ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/languagenavigation/languagenavigation.html

```diff
@@ -0,0 +1,14 @@
+<a class="language-navigation-btn mini-cart-shopping" href="javascript:void(0);">
```

Comment:
why we are using mini-cart-shopping class in language navigation component html

Reply:
@nitheshkumar-hcl  this is for common styling for header section.  @velhcl correct me if I am wrong.

Reply:
**mini-cart-shopping** is common class for header.
**language-navigation-btn** is the specific class for language navigation component